test/unit/signup/result_test.rb

```diff
@@ -105,7 +105,8 @@ class ResultTest < ActiveSupport::TestCase
       refute user.persisted?
       refute account.persisted?
       assert_includes signup_result.errors[:account], 'Organization/Group Name can\'t be blank'
-      assert_equal 1, signup_result.errors.full_messages.length
+      assert_includes signup_result.errors[:account], 'Organization/Group Name invalid'
+      assert_equal 2, signup_result.errors.full_messages.length
```

Comment:
maybe makes sense to validate the error message text also?

Reply:
Umm I have added both check we can remove also .I Have added so one should know like which are the two failing assertions.

Reply:
No I think because we have now added a custom message in en.yml and every time update that message this test will fail and we have to update here also which doesn't makes sense. 
I am updating and removing this assertion. Thanks for pointing this out.

Reply:
@mayorova changes done from my side

Reply:
But I don't think just checking the length of the errors is meaningful enough. What errors are they? Are they the ones we are expecting?

The fact that a change in `en.yml` make the test fail - I'd say it is good. If we accidentally or erroneously modify something in the i18n file - the test will capture it.

Reply:
I'm suggesting to add the following (fixing the current test and adding a new one):
```suggestion
    test '#save does not save and #errors return the errors when the account is blank' do
      @account = FactoryBot.build(:account, org_name: nil)
      signup_result.save
      refute user.persisted?
      refute account.persisted?
      assert_includes signup_result.errors[:account], "Organization/Group Name can't be blank"
    end

    test '#save does not save and #errors return the errors when the account is invalid' do
      @account = FactoryBot.build(:account, org_name: '?-!_$')
      signup_result.save
      refute user.persisted?
      refute account.persisted?
      assert_includes signup_result.errors[:account], "Organization/Group Name should contain at least one alphanumeric character"
    end
```

Reply:
I think writing two tests and checking for the two messages is OK, but then we would need to add tests for other fields like username and email. But maybe that's out of scope in this PR

Reply:
> But I don't think just checking the length of the errors is meaningful enough. What errors are they? Are they the ones we are expecting?
> 
> The fact that a change in `en.yml` make the test fail - I'd say it is good. If we accidentally or erroneously modify something in the i18n file - the test will capture it.
Agreed

Reply:
> I'm suggesting to add the following (fixing the current test and adding a new one):
> 
> ```
>     test '#save does not save and #errors return the errors when the account is blank' do
>       @account = FactoryBot.build(:account, org_name: nil)
>       signup_result.save
>       refute user.persisted?
>       refute account.persisted?
>       assert_includes signup_result.errors[:account], "Organization/Group Name can't be blank"
>     end
> 
>     test '#save does not save and #errors return the errors when the account is invalid' do
>       @account = FactoryBot.build(:account, org_name: '?-!_$')
>       signup_result.save
>       refute user.persisted?
>       refute account.persisted?
>       assert_includes signup_result.errors[:account], "Organization/Group Name should contain at least one alphanumeric character"
>     end
> ```

This can be also good idea

Reply:
> I think writing two tests and checking for the two messages is OK, but then we would need to add tests for other fields like username and email. But maybe that's out of scope in this PR

yes they are not something related to my change

Reply:
Updated so resolving